### PR TITLE
[18.0][MIG] l10n_ch_adr_report: Migration to 18.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**
### Depends on:

- [18.0][MIG] l10n_eu_product_adr_dangerous_goods: Migration to 18.0: `https://github.com/xaviedoanhduy/community-data-files/pull/1`

### Note

- Qty done and Reserved qty are merged into a single column in commit: `https://github.com/odoo/odoo/commit/7dda6bb92715ea25b2818a62fec5e646f3678b81`

### Result
[Swiss Dangerous Goods Report.pdf](https://github.com/user-attachments/files/17291330/Swiss.Dangerous.Goods.Report.pdf)

### CODE REVIEW

File: l10n_ch_adr_report/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+            if record.label_first:
+                res += f", {record._get_name_from_selection('label_first')}"
+
+            if record.label_second and record.label_third:
+                string_label_second = record._get_name_from_selection("label_second")
+                string_label_third = record._get_name_from_selection("label_third")
+                res += f", ({string_label_second}, {string_label_third})"
+            elif record.label_second:
+                res += ", ({})".format(record._get_name_from_selection("label_second"))
```
Comment (chaule97) [inline l10n_ch_adr_report/models/product_product.py]:
```suggestion
                res += f", ({record._get_name_from_selection('label_second')})"
```

File: l10n_ch_adr_report/readme/CREDITS.md [hunk 2]
```diff
@@ -1 +1,3 @@
 The development of this module was financially supported by Camptocamp
```
Comment (chaule97) [inline l10n_ch_adr_report/readme/CREDITS.md]:
```suggestion

```

Comment (xaviedoanhduy) [reply l10n_ch_adr_report/readme/CREDITS.md]: I think we should not remove that line, you can have a look on another modules are include above line development and migration like this `https://github.com/OCA/stock-logistics-workflow/blob/16.0/delivery_total_weight_from_packaging/readme/CREDITS.rst`

**Review by chaule97 — APPROVED** [review]

**Comment by xaviedoanhduy** [comment]
hi @chaule97, thanks for your time

**Pull request closed without merging**